{"name":"posix::error::errno::errno_location","from_trait":false,"arguments":[],"sources":[{"loc":"file: Real(\"src/posix/error.rs\") line 78","kind":{"UnsafeFnCall":"C"}}]}
